README.md

```diff
@@ -0,0 +1,177 @@
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 The Sky <a name="about-project"></a>
+
+**The Sky** A weather forecast app built with `React JS`, It has a cool design/UI and it shows the weather based on user's location, It also has the search functionality for searching for a specific city's weather condition and forecast..
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+   <li>HTML</li>
+   <li>CSS</li>
+   <li>Javascript</li>
+   <li>React Js</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Node js</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Clean Design**
+- **Responsive**
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live](https://the-sky.onrender.com/)
+
+## Watch the walkthrough here:
+
+- [Loom Walkthrough](https://www.loom.com/share/77484c8ca17742e6a639bd240fc44ab2)
```

Comment:
- [ ] Kindly consider recording a video and make sure the video contains a highlight about a piece of the code. You can consider the following checks when recording the video presentation :
     - [ ] Record a video presenting the features of the project.
     - [ ] The maximum length of the video is 5 minutes.
     - [ ] The video contains a clear and objective description of the project in professional English
     - [ ] The video contains a demo of the project, highlighting all the project features.
     - [ ] The video contains a highlight about a piece of the code that find interesting or is proud of making.

Reply:
@omarsalem7 Thank you very much for your kind review, but I have already recorded the video and I think I have explained all points that you listed, can you please be more specific on which points are missing on that video?

Reply:
Hi @BahirHakimy,
I hope you are doing well, but you forget to highlight a piece of code that made you proud of it by sharing a screen with your code in the IDE. you can refer to these requirements at [this link in project documentation](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=you%20should%20also%20highlight%20some%20interesting%20piece%20of%20code%20or%20something%20you%20built%20that%20you%20are%20very%20proud%20of.)
Are you need more explanations?